src/Orleans/ActivationService/ActivationServiceUtilities.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+
+namespace Orleans
+{
+    public static class ActivationServiceUtilities
+    {
+        private const string ActivationServiceNamesItemName = "ActivationServiceNamesItemName";
+        private const string ConstructorParametersItemName = "ConstructorParameters";
+
+        public static ParameterInfo BindToConstructorParameter(IGrainActivationContext grainActivationContext, Type activationServiceType)
+        {
```

Comment:
Minor. The method name confused me a little. I think the method is doing `BindActivationServiceTypeToGrainActivationContext`, or shorter `BindToActivationContext` in which the input params can come in and fill in the rest meaning. I don't have strong opinion on this. Just want to point out that it is a bit confusing and it may confuse other maintainer in the future.